javagen/src/main/java/com/azure/autorest/mapper/ClientMapper.java

```diff
@@ -327,13 +330,42 @@ public Client map(CodeModel codeModel) {
                 }
             };
 
-            syncClients.stream().filter(c -> c.getServiceClient() != null)
+            // protocol examples
+            syncClients.stream().filter(c -> c.getServiceClient() != null && CoreUtils.isNullOrEmpty(c.getConvenienceMethods()))
```

Comment:
Should we filter it in a per-method manner?

Reply:
Yeah, it could be a problem. We likely only want to generate example for client method with max overload. Previously it is done to filter for method with `Context`. But we didn't add this to convenience method.

Reply:
Got it. Maybe use the one with the longest parameters size?